lib/Mage/System/Ftp.php

```diff
@@ -211,7 +211,7 @@ public function getcwd()
      * ftp_raw wrapper
      *
      * @param string $cmd
-     * @return mixed
+     * @return array
```

Comment:
https://www.php.net/manual/en/function.ftp-raw.php
> Returns the server's response as an array of strings, or null on failure. No parsing is performed on the response string, nor does ftp_raw() determine if the command succeeded. 
```suggestion
     * @return string[]|null
```